### ISSUE TITLE
Is it Python 3

### ISSUE THREAD

**Issue by Sandy4321** [pmixer/SASRec.pytorch#2]
Original is Python 2
What is very limiting

**Comment by pmixer** [comment]
> Original is Python 2
> What is very limiting

yes, it's in python3, as we are only using a minimal set of python, if `print` and `division` got modified correctly, the original tensorflow version should also be fine to run in python3.
